### ISSUE TITLE
Multiple viewports in viewer

### ISSUE THREAD

**Issue by psarahdactyl** [libigl/libigl#1187]
I heard that there was support added for multiple viewports in a window using the libigl viewer. Is there a tutorial or example that I can see?

See <https://libigl.github.io/CONTRIBUTING/#bugreport> for more tips.

**Comment by jdumas** [comment]
If you're on the `dev` branch, take a look at tutorial 108. We're still missing a tutorial entry on the website for this.

**Comment by psarahdactyl** [comment]
Thank you!

**Comment by psarahdactyl** [comment]
Do you know how I would overlay one core over another? I was wondering if i could persistently have one be on top of the other.

**Comment by jdumas** [comment]
You mean like a miniature view over a main background view? We haven't really tested this use case, but as far as drawing goes it should work if the background one is rendered first. You may need to adjust the viewer behavior regarding how the mouse interactions are captured for each viewport though.

**Comment by psarahdactyl** [comment]
Right, exactly. oh okay, I see. so like camera controls for instance, I'd have to change for the miniature view?

**Comment by jdumas** [comment]
Yes. Have a look at the `Viewer::select_hovered_core()` function for example. Maybe a simple solution would be to reverse the order of the loop, so that viewports are drawn first to last, but selection of hovered viewport is done last to first.

**Comment by psarahdactyl** [comment]
how do I specify different callbacks for each core?

**Comment by jdumas** [comment]
In the callback you attach to the viewer, you could use `select_hovered_core()` and branch depending on which core is hovered by the mouse?
